keyboards/crkbd/keymaps/gotham/keycodes.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 3
```

Comment:
Would prefer to see this as an enum, as it's easier to maintain, long term. 

```suggestion
enum layers {
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```